### PR TITLE
Initial tests for extract script

### PR DESCRIPTION
**Pull request by indiah444**
- Very basic initial tests for `extract_fn.py` (more will have to be added later, along with a coverage report).
- Conftest file created for storing pytest fixtures.
- Requirements file updated to include necessary libraries.

### CODE REVIEW

File: fox_news_scraper/test_extract_fn.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# pylint: skip-file
```
Comment (megansam5) [inline fox_news_scraper/test_extract_fn.py:1]: Wow, weak!

File: fox_news_scraper/test_extract_fn.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+from unittest.mock import patch, MagicMock
+import feedparser
```
Comment (megansam5) [inline fox_news_scraper/test_extract_fn.py:4]: Not major important, just personal preference, but I do ones you don't need to pip install (unittest..) then space, ones you need to pip install (feedparser) then space, then all your own functions (extract stuff)

**Review by megansam5 — APPROVED** [review]
All looks good and a good basis to go off! <3

File: fox_news_scraper/test_extract_fn.py (hunk 1 above)
Comment (indiah444) [reply fox_news_scraper/test_extract_fn.py:1]: Haha, good point :(

File: fox_news_scraper/test_extract_fn.py (hunk 2 above)
Comment (indiah444) [reply fox_news_scraper/test_extract_fn.py:4]: Good point, I'll make that change now!

**Pull request merged**